activerecord/lib/active_record/connection_adapters/abstract/connection_pool.rb

```diff
@@ -986,15 +986,30 @@ def self.discard_unowned_pools(pid_map) # :nodoc:
         end
       end
 
+      attr_accessor :prevent_writes
```

Comment:
We don't need public `attr_writer :prevent_writes`.

```suggestion
      attr_reader :prevent_writes
```